### ISSUE TITLE
To top component icon sometimes does not get marked as loaded on loading for the first time in a session after browser start

### ISSUE THREAD

**Issue by Ambient-Impact** [Ambient-Impact/drupal-ambientimpact-ux#3]
I.e. the `ambientimpact-icon--icon-standalone-loaded` doesn't get applied to it. This could be an issue with the icon component or something else to do with load and attach timing. The whole icon component needs to be rewritten at some point, so this could be a symptom of that ancient JavaScript. This might also be a good opportunity to make the to top styling more resilient so it handles this in a less broken-looking way.

In short, you see the icon jumbled in with the text like so:
![Screenshot_20240326_081556](https://github.com/Ambient-Impact/drupal-ambientimpact-ux/assets/2289006/cec3bc83-397a-49c9-85bc-1bf48c810d50)

Whereas you should be seeing just the icon:
![Screenshot_20240326_081607](https://github.com/Ambient-Impact/drupal-ambientimpact-ux/assets/2289006/f8fd951f-f557-49bd-91b7-b24719fda70d)
